phoenicis-javafx/src/main/java/org/phoenicis/javafx/collections/ConcatenatedList.java

```diff
@@ -0,0 +1,351 @@
+     */
+    public ConcatenatedList(ObservableList<? extends ObservableList<E>> source) {
+        super(source);
+
+        this.expandedValues = source.stream().map(ArrayList::new).collect(Collectors.toList());
+
+        source.forEach(this::addUpdateListener);
+        fireChange(new InitialisationChange<>(0, size(), this));
+    }
+
+    /**
+     * Creates a new {@link ConcatenatedList} concatenating the given prefix values and the given {@link ObservableList
+     * list}
+     *
+     * @param list The list
+     * @param prefixes The prefix values
+     * @param <F> The instance type of the elements in the list and the prefix values
+     * @return A new {@link ConcatenatedList} containing all elements in given list and the prefix values
+     */
+    @SafeVarargs
+    public static <F> ConcatenatedList<F> createPrefixList(ObservableList<F> list, F... prefixes) {
+        return new ConcatenatedList<>(FXCollections.observableArrayList(
+                ImmutableList.<ObservableList<F>> builder()
+                        .add(FXCollections.observableArrayList(prefixes))
+                        .add(list).build()));
+    }
+
+    /**
+     * Creates a new {@link ConcatenatedList} concatenating the given {@link ObservableList list} and the given suffix
+     * values
+     *
+     * @param list The list
+     * @param suffixes The suffix values
+     * @param <F> The instance type of the elements in the list and the suffix values
+     * @return A new {@link ConcatenatedList} containing all elements in given list and the suffix values
+     */
+    @SafeVarargs
+    public static <F> ConcatenatedList<F> createSuffixList(ObservableList<F> list, F... suffixes) {
+        return new ConcatenatedList<>(FXCollections.observableArrayList(
+                ImmutableList.<ObservableList<F>> builder()
+                        .add(list)
+                        .add(FXCollections.observableArrayList(suffixes)).build()));
+    }
+
+    /**
+     * Creates a new {@link ConcatenatedList} with the given {@link ObservableList[] lists}
+     *
+     * @param lists The lists, which should be concatenated
+     * @param <F> The instance type of the elements in the to be concatenated lists
+     * @return A new {@link ConcatenatedList} containing all elements in the given lists
+     */
+    @SafeVarargs
+    public static <F> ConcatenatedList<F> create(ObservableList<F>... lists) {
+        return new ConcatenatedList<>(FXCollections.observableArrayList(lists));
+    }
+
+    /**
+     * Creates a new {@link ConcatenatedList} with the given {@link List[] lists}
+     *
+     * @param lists The lists, which should be concatenated
+     * @param <F> The instance type of the elements in the to be concatenated lists
+     * @return A new {@link ConcatenatedList} containing all elements in the given lists
+     */
+    @SafeVarargs
+    public static <F> ConcatenatedList<F> create(List<F>... lists) {
```

Comment:
It is a convenience method.
While the constructor requires a single `ObservableList` as its input parameter the `create` method takes multiple parameters.
I.e. the following call is possible: `ConcatenatedList.create(list1, list2, list3)`

Reply:
I understand what you mean, but this is just a helper method to avoid writing:
`ConcatenatedList.create(FXCollections.observableArrayList(list1, list2, list3))`